NOTICE-binary

```diff
@@ -1090,3 +1090,12 @@ decompression for Java, which can be obtained at:
     * license/LICENSE.lz4-java.txt (Apache License 2.0)
   * HOMEPAGE:
     * https://github.com/lz4/lz4-java
+
+This product optionally depends on 'Eclipse Collections', the Java collections framework
+ever that brings happiness to your Java development.
+It can be obtained at:
+  * LICENSE:
+    * licenses/LICENSE-EDL-1.0.txt (Eclipse Distribution License 1.0 (BSD))
+    * licenses/LICENSE-EPL-1.0.txt (Eclipse Public License 1.0)
+  * HOMEPAGE:
+    * https://eclipse.dev/collections/index.html
```

Comment:
nit: new line at end

Reply:
Actually, about the LICENSE information of third-party dependencies mentioned within the `NOTICE-binary` file, I have two points of confusion:

1. For instance, for 'lz4-java', its LICENSE information points to 'license/LICENSE.lz4-java.txt', it looks like a file path, but this file is not actually found in the project directory. I am unsure whether this is an error or if there is some other meaning behind it.

2. I have noticed that there are two folders related to licenses at the root of the project: `licenses` and `licenses-binary`, and I am unclear about what types of license files each of these folders should contain. 

I would appreciate clarification on how the community differentiates and considers these two directories.